iron-dropdown.html

```diff
@@ -207,10 +209,7 @@
          * Overridden from `IronOverlayBehavior`.
          */
         _renderClosed: function() {
-          if (!this.noAnimations && this.animationConfig && this.animationConfig.close) {
-            if (this.withBackdrop) {
-              this.backdropElement.close();
-            }
+          if (!this.noAnimations && this.animationConfig.close) {
```

Comment:
This is done in [`iron-overlay-manager`](https://github.com/PolymerElements/iron-overlay-behavior/blob/master/iron-overlay-manager.html#L229) for all the overlays.